_spinnaker/terraform_integration.md

```diff
@@ -198,7 +99,66 @@ Terraform's primary interface for user feedback is logging. When executed on you
 View the logs <a href="https://your-gate-url/proxies/terraform/api/v1/job/${#stage('Plan')['context']['status']['id']}/logs">here</a>
 ```
 
-
 ## Reference pipeline
 
 A reference pipeline which uses this feature can be found [here](https://gist.github.com/ethanfrogers/5123a5336f7e6ae4fd5fcda76536199b). It should help you get started! To use it, simply create a pipeline in the UI and click "Edit as JSON" under the "Pipeline Actions" dropdown and past the pipeline JSON into the text box.
+
+
+## Under the hood
+
+At the core of the Terraform integration is the Terraformer service. This service is reponsible for fetching your Terraform projects from source and exeuting various Terraform commands against them. When a `terraform` stage starts, Orca will submit the task to Terraformer and monitor it until completion. Once a task is submitted, Terraformer will fetch your target project, run `terraform init` to initialize the project and then run your desired `action` (`plan` or `apply`). If the task is successful, the stage will marked successful as well. If the task fails, the stage will be marked as a failure and halt the pipeline. 
```

Comment:
Speaking about Reltio they have some constraints for terraform repository structure, in which terraform commands need to be run with some extra parameters:
```
terraform get --update=true
terraform init -backend-config={somewhere inside the repo}
terraform plan -var-file={somewhere inside the repo}
terraform apply -var-file={somewhere inside the repo}
terraform output
```
Does terraformer supports these commands?